protocols/raft/src/main/java/io/atomix/protocols/raft/roles/LeaderRole.java

```diff
@@ -170,12 +172,15 @@ private void takeLeadership() {
   /**
    * Starts sending AppendEntries requests to all cluster members.
    */
-  private void startAppendTimer() {
+  private void startTimers() {
     // Set a timer that will be used to periodically synchronize with other nodes
     // in the cluster. This timer acts as a heartbeat to ensure this node remains
     // the leader.
-    log.trace("Starting append timer");
+    log.trace("Starting append timer on fix rate of {}", raft.getHeartbeatInterval());
     appendTimer = raft.getThreadContext().schedule(Duration.ZERO, raft.getHeartbeatInterval(), this::appendMembers);
```

Comment:
We are still regularly sending out append requests as well on top of the heartbeats? Do we need to do this? :thinking:

Reply:
Yes since appends are also kind of heartbeats. I would like to keep that to I think it makes it more stable then without.

Reply:
Fair enough, we can try without it at a later point